Github/manifest.json

```diff
@@ -33,5 +33,5 @@
     "name": "Github",
     "uuid": "7a951812-28a2-445a-a257-f26deae34d44",
     "slug": "github",
-    "version": "1.9.0"
+    "version": "1.9"
```

Comment:
for my understanding, is `1.9.0 < 1.9` ? do we need to change this version number if only the poetry.lock is updated ?

Reply:
Indeed, I have modified it.